README.md

```diff
@@ -27,7 +27,13 @@ buffer names.
 
 ### Installation
 
-Install the `ivy` package from MELPA / GNU ELPA.
+Install the `counsel` package preferably from MELPA. If you prefer to
+install from GNU ELPA, then make sure that the depenedencies -i.e. the
+`Ivy` and `Swiper` packages- are installed from GNU ELPA also (to
+prevent issue #2591).
```

Comment:
I don't think bug numbers belong under installation instructions of an active project.